hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/functional/TestConsistentBucketIndex.java

```diff
@@ -113,7 +113,7 @@ private void setUp(boolean populateMetaFields, boolean partitioned) throws Excep
             .withIndexType(HoodieIndex.IndexType.BUCKET)
             .withIndexKeyField("_row_key")
             .withBucketIndexEngineType(HoodieIndex.BucketIndexEngineType.CONSISTENT_HASHING)
-            .withBucketNum("8")
+            .withBucketNum("4")
```

Comment:
8 buckets across 3 partitions was causing some unnecessary overhead. This helps improve test runtime